CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## 0.9.0 - 2021-11-11
+
+### Added
+- Dynamic debugging to easily debug your static web app using the [Azure Static Web Apps CLI](https://github.com/Azure/static-web-apps-cli) [#557](https://github.com/microsoft/vscode-azurestaticwebapps/pull/557)
+
+    ![Debug static web app with a dynamic configuration](resources/changelog/debugging.gif)
```

Comment:
The .gif is very large (9.5 MB), and they don't compress well so it will significantly increase the VSIX size. In the extension page in VSCode, images in the readme get loaded directly from the web and so can be excluded from the VSIX via the `.vscodeignore` file, however I'm not sure if the same is true for the changelog.

I'd add this .gif (and the other .png image too) into the `.vscodeignore`, and check whether the image shows up when viewing the changelog from within the extension page. If it doesn't show up, I'd just remove it from the changelog, leaving it only in the readme where we know it will show up without needing to be in the VSIX.